test/backend/routes.spec.js

```diff
@@ -330,16 +326,17 @@ describe('Authentication - ', () => {
         // set access-token expiry of 3 sec:
         saveSetting('ACCESS_TOKEN_AGE', 3);
 
+        // TODO Are `res.json()` calls below even necessary? Checked response status is available without anyways...
```

Comment:
yeah - it does not look like `res.json` is needed here

Reply:
although we really should be checking the response, even if it is `{}` (like in https://github.com/plotly/falcon-sql-client/pull/212/files#diff-4565bfbd608d63bb50606edff170984cR465). @tarzzz - it looks like you wrote this test, could you update this test?